### ISSUE TITLE
Won't play Blu-ray ISO's over SMB (but DVD ISO's work)

### ISSUE THREAD

**Issue by kris-anderson** [newloran2/emplexer#47]
I have my emplexer.log file on hand, but I don't see an email I can send this to and GitHub won't allow me to attach it to this issue.

I have Plex running on my Ubuntu 12.04 Server using 172.30.0.61, and that server mounts the SMB shares from my unRAID server on 172.30.0.62. This allows my Plex Server running on Ubuntu to access my unRAID storage server. Everything is working well so far.

On my Dune, I changed the default Emplexer settings so that is uses SMB instead of HTTP. I set it to use the SMB of my unRAID server. It's playing everything I've thrown at it so far except for Blu-ray ISO files. These files are not encrypted, and they play fine when I access them using the Network Browser built into the Dune. But for some reason, they won't play when using Emplexer. When I click on the file the screen goes black and the loading icon appears in the top left. The Dune display says "WAIT" and after maybe 15 seconds I'm brought back to the Emplexer movies screen with no sort of warning or error message.

Is this a configuration issue on my end? Is there somewhere I can send my log file to help troubleshoot? Emplexer is playing my MKV's and normal DVD ISO's without any issues... so far this only seems related to Blu-ray ISO's.

**Comment by newloran2** [comment]
Hello kris-anderson,

I need to know some infos.
Which version of your dune firmware? the last version (130429_1605_b6) has a bug on bluray playback, if you use this version please try to downgrade an try again. 
If possible please do you can post your emplexer.log in a place like http://pastebin.com/ or dropbox or any other and post here the link?

**Comment by kris-anderson** [comment]
That was the problem. I updated to the latest beta on the Dune because I thought I read it was required for Emplexer. But once I downgraded to the previous version I have been using for months everything is working normally.

Thanks for the quick response. By the way, I love this project. About a year ago I was using My Movies and a whole bunch of WIndows PC's to manage everything. I've since switched to a Mac as my main machine, Linux Servers, and Dune's instead of Media Centers. This setup has been flawless, but it's always required my to manually manage my movies by launching Windows as a virtual machine on my Mac, and I had to navigate by SMB folder structure on the Dune to find my TV shows. Now that I have Plex running on my Ubuntu server, and Emplexer on my Dune, all my TV and Movies are automated and they have a slick interface to boot on the Dune.

Very happy right now... thank you. Keep up the great work and if I'm able to contribute to this project in any way I'll see what I offer.
